### ISSUE TITLE
add bitminter.com support

### ISSUE THREAD

**Issue by jakubwolny** [jakubwolny/mining-monitor#12]

**Comment by djmagicmetz** [comment]
Yeah, I'm on bitminter too. Looks like you support it, but all I see is a '0' for hashrate. When I click the coin icon it says 'error' under hashrate.

**Comment by mboehler** [comment]
It must have been fixed because bitminter works fine for me.
